resources/js/contest-voting/entry.coffee

```diff
@@ -61,11 +61,19 @@ export class Entry extends React.Component
       if @props.contest.show_votes
         div className: 'contest-voting-list__title contest-voting-list__title--show-votes',
           div className: 'contest-voting-list__votes-bar', style: { width: "#{relativeVotePercentage}%" }
-          div className: 'u-relative u-ellipsis-overflow', entry_title
+          div className: 'u-relative u-ellipsis-overflow',
+            a
+              className: 'contest-voting-list__title--link',
```

Comment:
in [BEM](https://getbem.com/naming/) the "modifiers" (`--link`) are for differetn versions of an element, but here it's something below `title` and not a variation of it. so i think this can be `contest-voting-list__title-link` instead, and put `&__title-link { ... }` at the same level as `&__title` in the css